### PR TITLE
Animating background color on mouse move

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: Components/MouseAnimator.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  hook = () => {
+    const body = document.body;
+
+    body.addEventListener('mousemove', (event) => {
+      const mouseXRatio = event.clientX / body.clientWidth;
+
+      body.style.setProperty("--bg-color", this.currentColor(mouseXRatio));
+    });
```
Comment (joaojusto) [inline Components/MouseAnimator.js]: Shouldn't we use a debouncer here?

File: css/Components/Root.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@import "../Theme/Color";
+
+body {
+  --bg-color: $Color-deep-cove;
```
Comment (joaojusto) [inline css/Components/Root.scss]: We could probably rename this variable into `Color-bg-color`; Or maybe we can move all of this into `Layout` and use `Layout-bg-color`;

**Comment by joaojusto** [comment]
@naps62 just take a look at my comments and :shipit:

File: css/Components/Root.scss (hunk 2 above)
Comment (naps62) [reply css/Components/Root.scss]: I don't agree with naming it after a Suit component. that assumes you're following Suit rules for this, which is not the case. You'd end up using a `Layout-bg-color` variable inside the Hero component

File: Components/MouseAnimator.js (hunk 1 above)
Comment (naps62) [reply Components/MouseAnimator.js]: it's actually a throttle. and it's done now

**Pull request merged**